pkg/network/setup/filters_test.go

```diff
@@ -47,4 +49,141 @@ var _ = Describe("Network setup filters", func() {
 			Expect(network.FilterNetsForVMStartup(vmi)).To(Equal([]v1.Network{*v1.DefaultPodNetwork()}))
 		})
 	})
+
+	Context("FilterNetsForLiveNICChange", func() {
+		const (
+			net1Name = "net1"
+			net2Name = "net2"
+			nad1Name = "nad1"
+			nad2Name = "nad2"
+		)
+
+		multusAndDomainInfoSource := vmispec.NewInfoSource(vmispec.InfoSourceMultusStatus, vmispec.InfoSourceDomain)
+
+		It("Should return an empty list when there are no networks to hot plug/unplug", func() {
+			vmi := libvmi.New(
+				libvmi.WithInterface(*v1.DefaultBridgeNetworkInterface()),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding(net1Name)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding(net2Name)),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithNetwork(libvmi.MultusNetwork(net1Name, nad1Name)),
+				libvmi.WithNetwork(libvmi.MultusNetwork(net2Name, nad2Name)),
+				libvmistatus.WithStatus(
+					libvmistatus.New(
+						libvmistatus.WithInterfaceStatus(v1.VirtualMachineInstanceNetworkInterface{
+							Name:       "default",
+							InfoSource: vmispec.InfoSourceDomainAndGA,
```

Comment:
nit: 
Some interfaces specify info-source with GA while other doesnt.
If we teat unit-tests as some sort of documentation source, its confusing because it implies the GA doesnt report for all NICs.

Please consider aligning all interfaces info-source to have GA or not, and avoid mixing.
This is true for other tests this PR adds.

Reply:
Removed all traces of GA, as it doesn't matter for this test.